platform/core/src/services/HangingProtocolService/lib/validator.js

```diff
@@ -1,20 +1,23 @@
 import validate from 'validate.js';
 
 validate.validators.equals = function (value, options, key, attributes) {
-  if (options && value !== options.value) {
-    return key + 'must equal ' + options.value;
+  const testValue = options?.value ?? options;
```

Comment:
Use the raw options value to make specifying the comparison easier - for example:
equals: 5
rather than:
equals: { value: 5 },